src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Product.xml

```diff
@@ -31,27 +30,55 @@
 
         <collectionOperations>
             <collectionOperation name="get">
+                <attribute name="path">admin/products</attribute>
                 <attribute name="filters">
                     <attribute>sylius.api.product_order_filter</attribute>
                     <attribute>sylius.api.product_taxon_code_filter</attribute>
                     <attribute>sylius.api.translation_order_filter</attribute>
                 </attribute>
             </collectionOperation>
+
             <collectionOperation name="post">
+                <attribute name="path">admin/products</attribute>
                 <attribute name="denormalization_context">
-                    <attribute name="groups">product:create</attribute>
+                    <attribute name="groups">admin:product:create</attribute>
+                </attribute>
+            </collectionOperation>
+
+            <collectionOperation name="get_latest">
+                <attribute name="method">GET</attribute>
+                <attribute name="path">shop/products/latest</attribute>
+                <attribute name="normalization_context">
+                    <attribute name="groups">
+                        <attribute>admin:shop:product:read</attribute>
```

Comment:
We should rather create two separate groups here

Reply:
true, rebase went wrong :(